randovania/games/samus_returns/json_data/Surface - East.txt

```diff
@@ -556,7 +558,9 @@ Extra - asset_id: collision_camera_014
   * Extra - actor_name: Door009
   * Extra - actor_type: doorpowerpower
   > Tunnel to Upper Storage
-      Morph Ball
+      All of the following:
+          Morph Ball
+          Walljump (Beginner) or Climb Rooms Vertically (High Jump)
```

Comment:
Yeah, am not opposed, i'd just like it to be consistent.
My main argument for having every WJ be a trick, is due to the game (afaik) never actually expecting you to do it. See for example first room in surface, where it wants you to use missiles rather than WJ up. If that kind of WJ is trickless, then that means that the missiles are never required in that spot and that would feel very weird to me.